konghao/client/src/client.cpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by andykong on 22-6-23.
+//
+#include "ros/ros.h"
+#include "std_msgs/String.h"
+
+#include <sstream>
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "talker");
+    ros::NodeHandle n;
+    ros::Publisher chatter_pub = n.advertise<std_msgs::String>("chatter", 1000);
+    ros::Rate loop_rate(10);
+
+    int count = 0;
+    while (ros::ok())
+    {
+
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Redundant blank line at the start of a code block should be deleted.  [whitespace/blank_line] [2]